WeScanSampleProject/ImplementReviewImageViewController.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ImplementReviewImageViewController.swift
+//  WeScanSampleProject
+//
+//  Created by Chawatvish Worrapoj on 8/1/2563 BE.
+//  Copyright © 2563 WeTransfer. All rights reserved.
+//
+
+import UIKit
+
+class ImplementReviewImageViewController: UIViewController {
```

Comment:
Same as the previous comment

```suggestion
final class ReviewImageViewController: UIViewController {
```